### PR TITLE
feat(chore): strip/adjust comment headers via config

### LINKED ISSUE #2468: [build] version in semantic.css should be mentioned only once
# Feature Request

`semantic.css` file is built from many source files and all of them have version in a CSS comment. This is a feature request to remove such comments when merging into one file (ie. put version on top of the file only once) to allow easier diff.

## Screenshot (if possible)

image

also the build year should be removed:

image

in favor of `Copyright GitHub Contributors`

### PR DESCRIPTION
**Pull request by lubber-de**
## Description
This PR allows to adjust the comment headers via config settings in `semantic.json`

- top comment banner in semantic(.min).(css/js) either by providing a complete custom text in `banner` or adjust some of the variables in the default banner template (year,url,repository,version,title) (of course you could combine them if the custom banner also has lodash templates variables inside via `<%= version %>` 
- strip all top headers from each component inside the concatenated semantic.css/js (this is already the case in the minified version) via `stripHeaders: true` . I intentionally did not set this to true by default as other peoples projects might rely on that.

I also fixed error fetching in case a themes name is not existing

#### Inside semantic.json
```json
"banner": "/* A custom central FUI Banner !!!\n Awesome Release! \n*/",
"stripHeaders": true, 
"header": {
    "year": "ALWAYS 2099 :)",
    "title": "Fomantic-UI Pro",
    "repository": "Not yet available",
    "url": "https://fomantic-ui.com/pro"
}
```
## Testcase
Change your semantic.json accordingly and `yarn build`

## Closes
#2468

### CODE REVIEW

File: tasks/config/tasks.js [hunk 1]
```diff
@@ -87,11 +97,11 @@ module.exports = {
...
     /* Comment Banners */
     header: {
-      year       : (new Date()).getFullYear(),
-      title      : release.title,
-      version    : release.version,
-      repository : release.repository,
-      url        : release.url
+      year       : nullish(config.header.year,(new Date()).getFullYear()),
+      title      : nullish(config.header.title, release.title),
+      version    : nullish(config.header.version, release.version),
```
Comment (mvorisek) [inline tasks/config/tasks.js:102]: is this compatible with https://github.com/fomantic/Fomantic-UI/blob/1c80027fc4cfaa894f7acad48cba59d64c555816/tasks/config/tasks.js#L65 and other replaces?

Comment (lubber-de) [reply tasks/config/tasks.js:102]: Should be, those values are for the central banner. The regexes you are reffering to are for the existing headers in each component

**Review by exoego — APPROVED** [review]
LGTM

**Comment by lubber-de** [comment]
Docs added by https://github.com/fomantic/Fomantic-UI-Docs/pull/417

**Pull request merged**